tests/test_mux.py

```diff
@@ -1226,6 +1226,35 @@ def test_neighbor_miss(
                 expected_mac=mac if exp_result[REAL_MAC] else '00:00:00:00:00:00'
             )
 
+    def test_neighbor_miss_no_mux(
+            self, dvs, dvs_route, setup_vlan, setup_tunnel, setup,
+            setup_peer_switch, neighbor_cleanup, testlog
+    ):
+        config_db = dvs.get_config_db()
+        appdb = swsscommon.DBConnector(swsscommon.APPL_DB, dvs.redis_sock, 0)
+
+        test_ip = self.SERV1_SOC_IPV4
+        self.ping_ip(dvs, test_ip)
+
+        # no mux present, no standalone tunnel route installed
+        self.check_neighbor_state(dvs, dvs_route, test_ip, expect_route=False)
+
+        # setup the mux
+        config_db = dvs.get_config_db()
+        self.create_mux_cable(config_db)
+
```

Comment:
Should we add another `ping_ip` and `check_neighbor_state` *after* mux is setup && *before* set port state to standby?

Reply:
good point, updated.